### PR TITLE
Fix error in Popen call of ffmpeg

### PR DESCRIPTION
**Pull request by natir**
Hello all, first thank for work on twitchIO,

## Pull request summary

I just see a small error in this extension, on my fedora (python 3.10) the sound extension not work properly, ffmpeg detection does not work. 

This call will fail even if `ffmpeg` is present in PATH:
```
proc = subprocess.Popen("ffmpeg -version", stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

This code also fix the error:
```
proc = subprocess.Popen(["ffmpeg", "-version"], stdout=subprocess.PIPE, stderr=subprocess.PIPE, shell=True)
```

## Checklist

- [ ] If code changes were made then they have been tested.
    - [ ] I have updated the documentation to reflect the changes.
    - [ ] I have updated the changelog with a quick recap of my changes.
- [X] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

**Comment by IAmTomahawkx** [comment]
Hello, please make sure you've added an entry to the changelog!

**Comment by natir** [comment]
Hi sorry I made this pull request to fast.

I add some other change, ffmpeg is called with `.exe`several time. I replace `has_ffmpeg` by `ffmpeg_bin` a variable with name of ffmpeg binary or None.

**Comment by chillymosh** [comment]
@natir Please make sure you have run this with black and line length 120

File: twitchio/ext/sounds/__init__.py [hunk 1]
```diff
@@ -43,13 +43,20 @@
 
 AP = TypeVar("AP", bound="AudioPlayer")
 
-has_ffmpeg: bool
+ffmpeg_bin: str = None
```
Comment (chillymosh) [inline twitchio/ext/sounds/__init__.py:46]: A string can't be None, this should be Optional[str]

**Comment by natir** [comment]
It fix sorry

File: twitchio/ext/sounds/__init__.py (hunk 1 above)
Comment (natir) [reply twitchio/ext/sounds/__init__.py]: It's done.

**Pull request merged**